core/src/main/java/org/axonframework/eventhandling/AbstractEventProcessor.java

```diff
@@ -92,6 +92,11 @@ public Registration registerInterceptor(MessageHandlerInterceptor<? super EventM
         return () -> interceptors.remove(interceptor);
     }
 
+    @Override
+    public Set<MessageHandlerInterceptor<? super EventMessage<?>>> getInterceptors() {
```

Comment:
I'd rename this function to `getHandlerInterceptors()`, as `getInterceptors()` is quite ambiguous with us having handler and dispatch interceptors. I know the Event Processors only have Handler Interceptors, but nonetheless I'd like to make that deviation.

Reply:
I kept it consistent with the `registerInceptor()`, but I could definitely change it to `getHandlerInterceptors()` if we don't want to keep the naming the same.